lib/view/list_widget.dart

```diff
@@ -3,7 +3,7 @@ import 'dart:async';
 import 'package:collection/collection.dart';
 import 'package:lists/model/item.dart';
 import 'package:lists/model/list_model.dart';
-import 'package:flutter/material.dart';
+import 'package:flutter/material.dart' hide SearchBar;
```

Comment:
What is this change for?

Reply:
I believe the most recent version of flutter includes their own [`SearchBar`](https://api.flutter.dev/flutter/material/SearchBar-class.html) widget, the name of which conflicts with ours.

Reply:
It would be better to rename our widget so that it doesn't conflict with things from Flutter

Reply:
How about `SearchField`?

Reply:
Sounds good